src/modules/Remote.rb

```diff
@@ -135,18 +149,25 @@ def Read
         from: "any",
         to:   "list <map>"
       )
-      vnc_conf = Builtins.filter(xinetd_conf) do |m|
-        s = Ops.get_string(m, "service", "")
-        s == "vnc1" || s == "vnchttpd1"
+
+      xinetd_vnc1_enabled = false
+      xinetd_vnchttp1_enabled = false
```

Comment:
I am not so fluent in ruby, but the reason was to declare the current block so they can be accessed later. Without these lines they exist only in the inner do-end block, and attempt to access them later gives:

`NameError: undefined local variable or method `xinetd_vnc1_enabled'` at src/modules/Remote.rb:149